cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -241,9 +237,9 @@ void ThinClientPoolDM::init() {
 
 ThinClientPoolDM::~ThinClientPoolDM() {
   destroy();
-  _GEODE_SAFE_DELETE(m_locHelper);
-  _GEODE_SAFE_DELETE(m_stats);
-  _GEODE_SAFE_DELETE(m_manager);
+  _GEODE_SAFE_DELETE(m_locHelper)
+  _GEODE_SAFE_DELETE(m_stats)
+  _GEODE_SAFE_DELETE(m_manager)
```

Comment:
Admittedly, I don't know the current best-practice in C++, but I think this looks a little meh and leads me think the macro is wrong.  If the spurious semicolon bothers you, then I would change the macro to...
```
#define _GEODE_SAFE_DELETE(x) \
 do  {                        \
    delete x;                 \
    x = nullptr;              \
  } while (0)
```

Reply:
this was default settings in CLion, and me hitting `option+enter` as fast I could, so didn't look that closely at the change.  I can change it back pretty easily (and don't have strong feelings about it).  Strangely enough CLion considered it error with the note `Empty expression statement has no effect; remove unncessary ';' to silence this warning` 🤷‍♀️

Reply:
I agree that an empty semicolon is silly.  Again though, as someone who has been heavily tainted by C, I'd suggest altering the macro to be do/while(0) rather than deleting the semicolon at all the call sites.  Besides the aesthetic appeal of seeing statements terminated by semicolon, it would reduce the diff if a future change moved the logic into a function by the same name.

Reply:
sound reasonable to me!